website/docs/deployment.md

```diff
@@ -10,7 +10,7 @@ This section provides all the help you need to deploy and operate Hudi tables at
 Specifically, we will cover the following aspects.
 
  - [Deployment Model](#deploying) : How various Hudi components are deployed and managed.
- - [Upgrading Versions](#upgrading) : Picking up new releases of Hudi, guidelines and general best-practices.
+ - [Upgrading/Downgrading Versions](#upgrading--downgrading) : Picking up new releases of Hudi, guidelines and general best-practices.
```

Comment:
minor. 
we can make Upgrading and Downgrading as two diff bullets. 
and fix the desc for downgrade.